### PR TITLE
Bugfix/pass text field type

### PR DESCRIPTION
**Pull request by JvJefke**
## PR Checklist

This PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [x] A changelog entry has been added to CHANGELOG.md if necessary

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] react application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
1. The type prop is not passed to the input of a TextField
2. There is no option to set a custom className

Issue Number: #192 

## What is the new behavior?
1. The type prop will be passed to the input
2. There is a option to set a custom className

## Does this PR introduce a breaking change?

- [x] Yes
- [] No

## Other information

The type prop worked in version 4.8.1 but not in 4.8.2. This PR should fix it.

## Resolved issues
Resolves: #192

### CODE REVIEW

**Review by TriangleJuice — APPROVED** [review]

**Comment by JvJefke** [comment]
@TriangleJuice FYI: I changed the className type from `String` to `string`.

File: packages/form/src/TextField/TextField.js [hunk 1]
```diff
@@ -76,7 +79,7 @@ class TextField extends Component<Props> {
         <div className="a-input__wrapper">
           {iconleft ? <Icon name={iconleft}/> : null}
           {mask ? <InputMask {...this.props} /> :
-            <input id={id} type="text" autoComplete={autoComplete} disabled={disabled}
+            <input id={id} type={type} autoComplete={autoComplete} disabled={disabled}
```
Comment (stefanpante) [inline packages/form/src/TextField/TextField.js:82]: omdat je hier "text" door type vervangt zou ik in de destructuring van de props type default op "text" zetten

Comment (frankdsm) [reply packages/form/src/TextField/TextField.js:82]: Had dezelfde opmerking

**Review by frankdsm — CHANGES_REQUESTED** [review]

**Comment by stefanpante** [comment]
@JvJefke Ik was vergeten op submit review te klikken 🤦

**Comment by JvJefke** [comment]
@stefanpante en @frankdsm Ik heb de default type `text` toegevoegd!

File: packages/form/src/TextField/TextField.js [hunk 2]
```diff
@@ -54,11 +55,14 @@ class TextField extends Component<Props> {
...
       loading,
       qa,
-      type,
+      type = 'text',
       errorDescription,
+      className,
       ...extraProps,
     } = this.props;
 
+    console.log('type', type);
```
Comment (TriangleJuice) [inline packages/form/src/TextField/TextField.js:64]: Alleen deze mag nog weg ;-)

Comment (JvJefke) [reply packages/form/src/TextField/TextField.js:64]: Right 🙈

**Review by frankdsm — APPROVED** [review]

**Comment by JvJefke** [comment]
Introduced breaking change on `type` prop. The size should be set by a `size` prop so the `type` prop is clear for default html types.

**Review by stefanpante — APPROVED** [review]

**Pull request merged**